packages/graphql/tests/tck/tck-test-files/interface-relationships/read.test.ts

```diff
@@ -59,6 +59,63 @@ describe("Interface Relationships", () => {
             config: { enableRegex: true, jwt: { secret } },
         });
     });
+    test("Top level match", async () => {
+        const query = gql`
+            query {
+                actors(where:{actedInConnection:{node:{title:"The Matrix", _on:{Movie:{title:"Matrix"}}}}}) {
+                    actedIn {
+                        title
+                        ... on Movie {
+                            runtime
+                        }
+                        ... on Series {
+                            episodes
+                        }
+                    }
+                }
+            }
+        `;
```

Comment:
Can you please make sure that formatting has been applied? This query should look a lot nicer.

Reply:
done